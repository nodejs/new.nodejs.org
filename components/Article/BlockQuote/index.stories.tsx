import BlockQuote from './index';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

type Story = StoryObj<typeof BlockQuote>;
type Meta = MetaObj<typeof BlockQuote>;

export const Default: Story = {
  args: { children: 'This is a block quote' },
};

export const MultipleParagraph: Story = {
  args: {
    children: [
      <p key={1}>This is a block quote 1</p>,
      <p key={2}>This is a block quote 2</p>,
    ],
  },
};

export default { component: BlockQuote } as Meta;
