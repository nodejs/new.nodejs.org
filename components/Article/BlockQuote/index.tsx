import styles from './index.module.scss';
import type { FC, PropsWithChildren } from 'react';

const BlockQuote: FC<PropsWithChildren> = ({ children }) => (
  <div className={styles.blockQuote}>{children}</div>
);

export default BlockQuote;
