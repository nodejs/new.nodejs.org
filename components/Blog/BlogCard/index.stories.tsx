import BlockQuote from './index';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

type Story = StoryObj<typeof BlockQuote>;
type Meta = MetaObj<typeof BlockQuote>;

export const Default: Story = {
  args: {
    author: 'Bat Man',
    category: 'category-mock',
    date: '2023-04-21 23:40:56.77',
    slug: '/blog/category-mock/sample-blog',
    title: 'Sample Test Blog',
    readingTime: '1 min read',
  },
};

export default { component: BlockQuote } as Meta;
