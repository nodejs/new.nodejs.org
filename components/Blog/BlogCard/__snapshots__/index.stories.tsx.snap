// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog/BlogCard Default smoke-test 1`] = `
<div class="BlogCard_blogCard__BP5FM">
  <div class="BlogCard_title__eA5iq">
    <a href="/en/blog/category-mock/sample-blog">
      Sample Test Blog
    </a>
    <div class="BlogCard_metadata__dWJWM">
      <a class="BlogCard_category__yR5u3"
         href="/en/blog/category-mock"
      >
        category-mock
      </a>
      <span>
        1 min read
      </span>
    </div>
  </div>
  <div class="BlogCard_content__jnh1D">
    <h4>
      April 21, 2023
    </h4>
    <p>
      by
      <span>
        Bat Man
      </span>
    </p>
  </div>
</div>
`;
